### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by BrennanConroy**
Automated PR broken by https://github.com/dotnet/extensions/commit/fc7a9d580cd60d84bb4739122a41c3a4c44751dc

### CODE REVIEW

**Comment by sebastienros** [comment]
Looks like it also contains the changes made by darc here: https://github.com/dotnet/aspnetcore/pull/19549 which is blocked by a flaky test (quarantining PR currently building).

If we merge this one, can we close the other darc PR?

**Comment by BrennanConroy** [comment]
Yes

**Review by sebastienros — APPROVED** [review]

**Comment by sebastienros** [comment]
> If we merge this one, can we close the other darc PR?

@NTaylorMullen In case it's not done until tomorrow

**Comment by sebastienros** [comment]
Unfortunately the other darc PR got updated with newer versions, so it will still need to be processed.

**Comment by dougbu** [comment]
Recommend we merge this to minimize how out-of-date we are

File: eng/Version.Details.xml [hunk 1]
```diff
@@ -348,13 +348,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>8ccad075bbb0db445e03eed0a6073d27bdd4f31a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.AspNetCore.Testing" Version="5.0.0-preview.2.20151.1" CoherentParentDependency="Microsoft.AspNetCore.Razor.Language">
```
Comment (dougbu) [inline eng/Version.Details.xml]: Hurrah 🎆

File: eng/Versions.props [hunk 2]
```diff
@@ -64,95 +64,94 @@
...
-    <MicrosoftNETCoreAppInternalPackageVersion>5.0.0-preview.2.20129.8</MicrosoftNETCoreAppInternalPackageVersion>
-    <MicrosoftNETCoreAppRefPackageVersion>5.0.0-preview.2.20129.8</MicrosoftNETCoreAppRefPackageVersion>
-    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>5.0.0-preview.2.20129.8</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
-    <NETStandardLibraryRefPackageVersion>2.1.0-preview.2.20129.8</NETStandardLibraryRefPackageVersion>
+    <MicrosoftExtensionsDependencyModelPackageVersion>5.0.0-preview.2-runtime.20154.1</MicrosoftExtensionsDependencyModelPackageVersion>
+    <MicrosoftNETCoreAppInternalPackageVersion>5.0.0-preview.2.20154.1</MicrosoftNETCoreAppInternalPackageVersion>
+    <MicrosoftNETCoreAppRefPackageVersion>5.0.0-preview.2.20154.1</MicrosoftNETCoreAppRefPackageVersion>
+    <MicrosoftNETCoreAppRuntimewinx64PackageVersion>5.0.0-preview.2.20154.1</MicrosoftNETCoreAppRuntimewinx64PackageVersion>
+    <NETStandardLibraryRefPackageVersion>2.1.0-preview.2.20154.1</NETStandardLibraryRefPackageVersion>
     <!-- Packages from dotnet/corefx -->
```
Comment (dougbu) [inline eng/Versions.props]: nit: Could we remove the dotnet/corefx and dotnet/core-setup mentions in our 'master' branch in the automated PR update after this one goes in?

**Review by dougbu — APPROVED** [review]

**Pull request merged**